docs/_includes/default.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>{{ page.title }}</title>
+    <link
+      rel="preload"
+      href="https://opencollective.com/static/images/user.svg"
+      as="image"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/prism.css" />
+    <link rel="shortcut icon" href="favicon.ico" />
+
+    <!--[if lt IE 9]> <script src="js/html5shiv.min.js"></script> <![endif]-->
+  </head>
+
+  <body>
+    <header id="_header">
+      <h1>
+        <a href="/">
+          <img
+            src="/images/mocha-logo.svg"
+            alt="Mocha"
+            width="192"
+            height="192"
+          />
+        </a>
+      </h1>
+
+      <p id="tag"><em>simple</em>, <em>flexible</em>, <em>fun</em></p>
+    </header>
+
+    <main id="content">{{ content }}</main>
+
+    <footer>
+      <span>
+        <a href="https://mochajs.org">mochajs.org</a> is licensed under a
+        <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"
+          >Creative Commons Attribution 4.0 International License</a
+        >.
+        <p>
+          <em>Last updated: {{ 'now' | date: '%a %b %d %H:%M:%S %Y' }}</em>
+        </p></span
+      >
+    </footer>
+
+    <script src="js/ga.js" async></script>
```

Comment:
Tracking still? 
Think script's supposed to be loaded in "<head>" to be most effective

Reply:
It discussed in https://github.com/mochajs/mocha/issues/3349